src/main/java/com/fasterxml/jackson/databind/util/StdDateFormat.java

```diff
@@ -159,10 +164,11 @@ public StdDateFormat(TimeZone tz, Locale loc) {
         _locale = loc;
     }
 
-    protected StdDateFormat(TimeZone tz, Locale loc, Boolean lenient) {
```

Comment:
For  backwards compatibility we must leave the older constructor as well, maybe as deprecated.

Reply:
Actually I can make that change. I'll change setter to `withX()` as well, to conform to other mutators (minus existing `setLenient()` which can't be changed coming from `DateFormat`).